src/node_lttng.cc

```diff
@@ -248,7 +248,7 @@ void InitLTTNG(Environment* env, Local<Object> target) {
 #undef NODE_PROBE
   };
 
-  for (unsigned int i = 0; i < ARRAY_SIZE(tab); i++) {
+  for (unsigned int i = 0; i < arraysize(tab); i++) {
```

Comment:
I thought I caught them all (I updated them in other files) but I must've overlooked this file.  I'll fix that before landing.